src/lib/serviceRoutinesV2/getEntities.cpp

```diff
@@ -70,6 +70,11 @@ std::string getEntities
   string    idPattern  = ciP->uriParam["idPattern"];
   string    q          = ciP->uriParam["q"];
 
+  if (ciP->uriParam["options"] == "count")
```

Comment:
Yes, probably is misleanding, sorry...

What I tried to mean in #1041 is that in PR #958 "get all entities" operation was implemented using `count=true` and that that implementation needs to be fixed to use `count` option (not exclusively: I mean, it could happen that other operations appar from "get all entities" are currently using `count=true`, in that case, they also need to be fixed to use `count` option).

Bottom line: at the end only `count` option will be the valid way of getting the count in any NGSIv2 operation in which counting makes sense and this PR should address that (of course, only for currently implemented operations, e.g. list subscription is not).